### ISSUE TITLE
NVDA + Baum SuperVario braille display unable to get back from hibernation

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4851]
**Reported by msuch on 2015-01-26 17:34**
The problem occurs with Next Snapshots, did not occur with 2014.4.
When the user tries to wake her machine from hibernate mode, NVDA crashes and the only way to restart it is to restart the computer.
Here is a part of the nvda.log.
ERROR - queueHandler.flushQueue (16:24:55):
Error in func _handleUpdate from eventQueue
Traceback (most recent call last):
  File "queueHandler.pyc", line 50, in flushQueue
  File "virtualBuffers__init__.pyc", line 1262, in _handleUpdate
  File "braille.pyc", line 1475, in handleUpdate
  File "braille.pyc", line 774, in update
  File "textInfos\offsets.pyc", line 344, in expand
  File "virtualBuffers__init__.pyc", line 322, in _getUnitOffsets
  File "textInfos\offsets.pyc", line 326, in _getUnitOffsets
  File "virtualBuffers__init__.pyc", line 258, in _getLineOffsets
WindowsError: [1775](Error) Un descripteur de contexte null a été passé du client à l’hôte lors d’un appel de procédure distante (RPC)

**Comment by leonardder** [comment]
@MichelSuch: Could you provide a status update for this?

**Comment by leonardder** [comment]
Given that there are several similar issues around for other displays and that braille display auto detection has been implemented, I"m closing this.

**Comment by jcsteh** [comment]
IMO, auto detection was always the correct fix for this anyway, which is why I never looked at this in isolation. We shouldn't be fixing such issues on a per driver basis.
